### DISCUSSION TITLE
Cannot refresh auth when using cloudflare

### DISCUSSION THREAD

**Discussion by Inovvia** [pocketbase/pocketbase#5722] [Q&A] [answered]
Im trying to use the auth.refresh() but I keep getting errors on pocketbase. The method works when I'm accessing pocketbase directly but not when using cloudflare tunnels.

**Comment by Danjoe4** [comment]
Did you try setting cloudflare on dev mode and clearing your browser cache?

**Reply by Inovvia** [reply]
Yep nothing still happens.

**Reply by Danjoe4** [reply]
Please post the relevant code, tunnel configuration, and errors you're getting

**Reply by Inovvia** [reply]
https://github.com/pocketbase/pocketbase/discussions/5722#discussioncomment-11018597

**Comment by Inovvia** [comment]
Here is the code
https://github.com/pocketbase/pocketbase/issues/5721

I'm also using basic cloudflare tunnel and pocketbase via docker.

**Comment by Inovvia** [comment, marked as answer]
I have no idea why this happens on bare cloudflare 🤣.

To solve this here is what I did - use an nginx proxy and pointed cloudflare to nginx 🤣

**Comment by bUgESc5Ep** [comment]
I had the same problem. As it turns out cloudflare out of the box adds the "Transfer-Encoding: chunked" header to the request and this causes the error. 
You can disable this header in the [cloudflare tunnel settings](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/configure-tunnels/origin-configuration/#disablechunkedencoding).
After I disabled the chunked encoding in cloudflare auth.refresh() worked again.
